cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -44,9 +43,209 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
                     renderViewsSpy;
                 showModal();
                 renderViewsSpy = spyOn(modal, 'renderViews');
-                AjaxHelpers.expectRequest(requests, 'GET', OUTLINE_URL);
+                AjaxHelpers.expectRequest(requests, 'GET', outlineUrl);
                 AjaxHelpers.respondWithJson(requests, {});
                 expect(renderViewsSpy).toHaveBeenCalled();
             });
         });
+
+        showModal = function() {
+            modal = new MoveXBlockModal({
+                sourceXBlockInfo: new XBlockInfo({
+                    id: sourceLocator,
+                    display_name: sourceDisplayName,
+                    category: 'html'
+                }),
+                sourceParentXBlockInfo: new XBlockInfo({
+                    id: sourceParentLocator,
+                    display_name: 'VERT 101',
+                    category: 'vertical'
+                }),
+                XBlockUrlRoot: '/xblock',
+                outlineURL: outlineUrl
+            });
+            modal.show();
+        };
+
+        selectTargetParent = function(parentLocator) {
+            modal.moveXBlockListView = {
+                parent_info: {
+                    parent: {
+                        id: parentLocator
+                    }
+                },
+                remove: function() {}   // attach a fake remove method
+            };
+        };
+
+        getConfirmationFeedbackTitle = function(displayName) {
+            return StringUtils.interpolate(
+                gettext('Success! "{displayName}" has been moved to a new location.'),
+                {
+                    displayName: displayName
+                }
+            );
+        };
+
+        getUndoConfirmationFeedbackTitle = function(displayName) {
+            return StringUtils.interpolate(
+                gettext('Undo Success! "{sourceDisplayName}" has been moved back to a previous location.'),
+                {
+                    sourceDisplayName: displayName
+                }
+            );
+        };
+
+        getConfirmationFeedbackTitleLink = function(parentLocator) {
+            return StringUtils.interpolate(
+                gettext(' {link_start}Take me there{link_end}'),
```

Comment:
@muzaffaryousaf In this case, probably yes, :-) although there might be other strings I have not seen in context that might cause tweaks to be required. We wanted the link to read more completely "Take me to the new location" rather than "there".